lessons/env-and-tools/README.md

```diff
@@ -1 +1,12 @@
-# put instructions here
+# How To Download Xcode
+
+1. From your Macbook, open the App Store.
+2. Search for *Xcode*.
+..* Note: Make sure you’re downloading Version 7.3.1. Do not download Xcode Beta 8 at this point. 
+3. Press ‘Get’ and then ‘Install App’. 
+4. You will now be prompted to provide your AppleID. If you do not have one, create one. Once you’ve provided a correct AppleID, click Buy.
+..*Note: Xcode is free. You do not have to make any purchases to download and use Xcode.
+4. When download is complete, open *Xcode* from your Applications folder. 
+5. The first time you open *Xcode* it’ll ask to install some additional components, allow it.
+6. Once complete, you should see a “Welcome to Xcode” screen. 
+7. Congratulations! You have successfully downloaded Xcode. Now you’re able to do the [pre-work assignment] (https://github.com/C4Q/AC3.2/blob/master/lessons/prework)
```

Comment:
Jason - I added the readme for the pre-work as well as the Playground
exercise file

On Fri, Jul 15, 2016 at 4:23 PM, jgresh <notifications@github.com> wrote:

> In lessons/env-and-tools/README.md
> <https://github.com/C4Q/AC3.2/pull/5#discussion_r71035567>:
>
> > @@ -1 +1,12 @@
> > -# put instructions here
> > +# How To Download Xcode
> > +
> > +1. From your Macbook, open the App Store.
> > +2. Search for *Xcode*.
> > +..* Note: Make sure you’re downloading Version 7.3.1. Do not download Xcode Beta 8 at this point.
> > +3. Press ‘Get’ and then ‘Install App’.
> > +4. You will now be prompted to provide your AppleID. If you do not have one, create one. Once you’ve provided a correct AppleID, click Buy.
> > +..*Note: Xcode is free. You do not have to make any purchases to download and use Xcode.
> > +4. When download is complete, open *Xcode* from your Applications folder.
> > +5. The first time you open *Xcode* it’ll ask to install some additional components, allow it.
> > +6. Once complete, you should see a “Welcome to Xcode” screen.
> > +7. Congratulations! You have successfully downloaded Xcode. Now you’re able to do the [pre-work assignment] (https://github.com/C4Q/AC3.2/blob/master/lessons/prework)
>
> This link got in there: (
> https://github.com/C4Q/AC3.2/blob/master/lessons/prework)
>
> Is something doing this automatically?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/C4Q/AC3.2/pull/5/files/5a790b9ec3e09276f8f6d9ccc2b2646a7cc7605b#r71035567>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALU5My7Gf1mlMR8bWxd4_rLWXlhP46RAks5qV-xagaJpZM4JNnu9>
> .
>